### PR TITLE
new parser networkmanager_config

### PR DESCRIPTION
**Pull request by rasrivas-redhat**
Signed-off-by: rasrivas <rasrivas@redhat.com>

### CODE REVIEW

**Review by xiangce — APPROVED** [review]

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

File: insights/parsers/tests/test_networkmanager_config.py
Comment (bfahr) [inline insights/parsers/tests/test_networkmanager_config.py:63]: @rasrivas-redhat could you actually check the value here, and not just that its present in the data?  Also it might be good to have a negative test, perhaps that `[main] plugins` is not present?

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/tests/test_networkmanager_config.py
Comment (rasrivas-redhat) [reply insights/parsers/tests/test_networkmanager_config.py]: @bfahr - I have updated the test case for it and also added a test case to check for a negative test, please review.

File: insights/parsers/tests/test_networkmanager_config.py
Comment (bfahr) [inline insights/parsers/tests/test_networkmanager_config.py:112]: @rasrivas-redhat you need to add `assert` here.

File: insights/parsers/tests/test_networkmanager_config.py
Comment (bfahr) [inline insights/parsers/tests/test_networkmanager_config.py:117]: Same as comment above.

**Review by bfahr — CHANGES_REQUESTED** [review]

**Review by bfahr — APPROVED** [review]

**Pull request merged**
